index.html

```diff
@@ -211,7 +205,7 @@ <h2>Videos</h2>
 &lt;/section&gt;</code></pre>
             <figcaption>
               <h1>Code snippet of a video with a caption.</h1>
-              <p>As with images, videos can be inserted in the article header.</p>
+              <p>As with images, videos can be inserted in the article header and have a default width of 600px.</p>
```

Comment:
It seems that the 600 px default width is limited to videos inside the article header, while I suppose that was a general statement. You could reverse the order of the two propositions to make the sentence clearer.

Reply:
Ah! This sentence is really misleading, thanks to pointing that out.  I replaced it with:

> As with images, videos have a default width of 600px, customisable with inline declarations as described in section Customisations.